homework-2/index.ts

```diff
@@ -0,0 +1,9 @@
+import express, { Application } from 'express';
+import { userRouter } from "./routes/userRouter";
+import { APP_PORT } from "./config/constants";
+
+const app: Application = express();
```

Comment:
Please create http server and set it into express

Reply:
Done